x/ibcratelimit/README.md

```diff
@@ -1,4 +1,4 @@
-# ibcratelimit
+# x/ibcratelimit
```

Comment:
The others have this wrapped in ticks. It's also missing the `sidebar_position`.

The `sidebar_position` is missing from msgfees and ibchooks readmes too.